rest/index.conf

```diff
@@ -1,5 +1,5 @@
 [options]
 kibanaurl=http://localhost:9090/app/kibana
-esservers=http://localhost:9200
+esservers=http://elasticsearch:9200
```

Comment:
I don't know how I feel about us modifying main config files to work in docker (there were a couple other ones recently as well).

I think we should keep these files as they would be for a non docker environment, and then make local modifications if we need it to work in docker, or have docker overwrite these files accordingly.

Reply:
Yeah, good point. I didn't realize the rest docker container had it's own .conf. For whatever reason it wasn't working for me without this change, but I'd guess it's something in my local docker setup. I'll revert this.